### ISSUE TITLE
check if these transit peptides should be removed

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3727]
> good point, I suspect that there will not be many sequence changes for the ones with signal peptide/transit peptide.

I found these where there's a AA difference in the first 30 amino acids and there's a signal_peptide or transit_peptide:

- [x] SPAC13C5.03 signal_peptide see https://github.com/pombase/curation/issues/3739
- [x] SPAC227.11c signal_peptide Annotated
- [x] SPAPB24D3.10c signal_peptide annotated
- [ ] SPAC23C11.17 transit_peptide
- [ ] SPAC5H10.06c transit_peptide
- [ ] SPAC167.04 transit_peptide
- [ ] SPAC1B3.18c transit_peptide

_Originally posted by @kimrutherford in https://github.com/pombase/pombase-chado/issues/52#issuecomment-2265008914_

**Comment by ValWood** [comment]
I will look at these (these are sequences that lost transit peptides because of alterations. I want to double check

**Comment by ValWood** [comment]
I can't remember what I'm checking here?

**Comment by kimrutherford** [comment]
> I can't remember what I'm checking here?

The sequence is different (out of date) in UniProt but they have annotated signal or transit peptides.  These are cases where the sequence if different in the first 30 amino acids so the signal/transit peptide sequence is likely to change.

I checked the three signal peptide genes with SignalP.  It says SPAC13C5.03 doesn't have a signal peptide, SPAC227.11c probably does (likelihood 0.6665) and SPAPB24D3.10c very probably (likelihood 0.9998).

I looked up the 4 annotated transit peptides from the list on the TransitP site (https://services.healthtech.dtu.dk/services/TargetP-2.0/).

It predicts that SPAC167.04, SPAC1B3.18c and SPAC23C11.17 have a transit peptide.  But not SPAC5H10.06c.

Should I remove that annotations from SPAC13C5.03 and SPAC5H10.06c?
